pyjobshop/Model.py

```diff
@@ -127,7 +122,7 @@ def add_job(
         Job
             The created job.
         """
-        job = Job(weight, release_date, deadline, due_date, name)
```

Comment:
How to initialize tasks here?

Reply:
We also don't do this in PyVRP for ClientGroup. I think we don't need to do it here either.